README.md

```diff
@@ -1,5 +1,7 @@
 [![Circle CI](https://circleci.com/gh/compose/transporter.svg?style=svg)](https://circleci.com/gh/compose/transporter)
 
+Copmopse helps with database transportation and transformations from one store to another.
```

Comment:
small typo here, Copmopse -> Compose

also, transporter can sync from one to many stores but I'm fine leaving what you've put as is